transport-classes-io_uring/src/main/java/io/netty/channel/uring/SubmissionQueue.java

```diff
@@ -215,9 +230,9 @@ int submitAndWait() {
     }
 
     private int submit(int toSubmit, int minComplete, int flags) {
-        PlatformDependent.putIntOrdered(kTailAddress, tail); // release memory barrier
+        INT_HANDLE.setRelease(kTail, 0, tail);
         int ret = ioUringEnter(toSubmit, minComplete, flags);
-        head = PlatformDependent.getIntVolatile(kHeadAddress); // acquire memory barrier
+        head = (int) INT_HANDLE.getVolatile(kHead, 0); // acquire memory barrier
```

Comment:
Is `INT_HANDLE.getAcquire` feasible here?

Reply:
Also let's just replace things for now and not change them in way they might behave.